doc/userguide/rules/vlan-id.rst

```diff
@@ -0,0 +1,33 @@
+Vlan.id Keyword
+===============
+
+Suricata has a ``vlan.id`` keyword that can be used in signatures to identify
+and filter network packets based on Virtual Local Area Network IDs. By default,
+it matches all layers if a packet contains multiple VLAN layers. However, if a
+specific layer is defined, it will only match that layer.
+
+vlan.id uses :ref:`unsigned 16-bit integer <rules-integer-keywords>`.
```

Comment:
https://github.com/OISF/suricata/pull/12273#discussion_r1884368088

> Should we also mention here the max value, and the specification that defines that?

I think it is a good idea